src/main/java/src/WordleDictionary.java

```diff
@@ -0,0 +1,49 @@
+package src;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.*;
+
+public class WordleDictionary {
+    public ArrayList<String> words;
+    public ArrayList<String> definitions;
+
+    public WordleDictionary(String pathname) throws FileNotFoundException {
+        File file = new File(pathname);
+        words = new ArrayList<>();
+        definitions = new ArrayList<>();
+        Scanner dictionary = new Scanner(file);
+        String line = dictionary.nextLine();
+        while (true) {
+            int i = 0;
+            int idx = 0;
+            while (i < line.length()) {
```

Comment:
If you're going to do this, use a for loop.
You can also use `line.IndexOf(" ")`